modules/remove.js

```diff
@@ -0,0 +1,16 @@
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ] Kindly remove the eslint disable line, it is always better to fix all issues in the code.